src/Workspaces/Core/Portable/Workspace/Host/HostProjectServices.cs

```diff
@@ -6,16 +6,18 @@
 
 namespace Microsoft.CodeAnalysis.Host
 {
-    // TODO(cyrusn): Make public.  Tracked through https://github.com/dotnet/roslyn/issues/62914
-    internal sealed class HostProjectServices
+    /// <summary>
+    /// Per language services provided by the host environment.
+    /// </summary>
+    public sealed class LanguageServices
```

Comment:
New public API.   ANd all public members in this are the new public api shape.  It's worth expanding this file to see/understand them.